Podcast/PlayerControlsView.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  PlayerControlsView.swift
+//  Podcast
+//
+//  Created by Mark Bryan on 9/14/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class PlayerControlsView: UIView {
+    
+    // Mark: Constants
+    
+    let PlayerControlsViewHeight: CGFloat = 182
+    
+    let MoreButtonSize: CGFloat = 24
+    let ForwardsBackwardsButtonSize: CGFloat = 44
+    let PlayPauseButtonSize: CGFloat = 40
+    let SliderHeight: CGFloat = 6
+    
+    let SliderInset: CGFloat = 48
+    let SliderCenterY: CGFloat = 78
+    let MoreButtonYValue: CGFloat = 29
+    let MoreButtonHorizontalInset: CGFloat = 17
+    let EpisodeNameLabelYValue: CGFloat = 21
+    let SeriesNameLabelYValue: CGFloat = 40
+    let TimeLabelHorizontalInset: CGFloat = 14
+    let ControlButtonsCenterY: CGFloat = 120
+    let ControlButtonsCenterHorizontalOffset: CGFloat = 55
+    
+    // Mark: Properties
+     
+    var episodeNameLabel: UILabel!
+    var seriesNameLabel: UILabel!
+    var slider: UISlider!
+    var moreButton: UIButton!
+    var playPauseButton: UIButton!
+    var forwardsButton: UIButton!
+    var forwardsLabel: UILabel!
+    var backwardsButton: UIButton!
+    var backwardsLabel: UILabel!
+    var rightTimeLabel: UILabel!
+    var leftTimeLabel: UILabel!
+    
+    var player: Player?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = .whiteColor()
+        self.frame.size.height = PlayerControlsViewHeight
```

Comment:
I tried getting rid of self in `self.frame.size.height` and swift complains since frame is also a parameter in the init method (and it is a let constant there). I think this one needs to stay.